.travis.yml

```diff
@@ -1,14 +1,15 @@
 sudo: false
 language: go
 go:
-  - tip
+  - master
+  - 1.9
+  - 1.8
   - 1.7
   - 1.6
 before_script:
-  - go get -u github.com/golang/lint/golint
-install:
-  - go get github.com/Masterminds/glide
-  - glide install
+  - cd vendor/github.com/golang/lint/golint
+  - go install
+  - cd -
```

Comment:
We can vendor `golint` by putting it in the `required` section of `Gopkg.toml`. That's one less network request and one less reason for CI to fail.